src/database/utilities/parsers.js

```diff
@@ -5,13 +5,11 @@
 
 /**
  * Returns the boolean string as a boolean (false if none passed)
- * @param  {string} numberString The string to convert to a boolean
+ *
+ * @param  {string}  numberString  The string to convert to a boolean
  * @return {boolean}               The boolean representation of the string
  */
-export const parseBoolean = booleanString => {
-  const trueStrings = ['true', 'True', 'TRUE'];
-  return trueStrings.includes(booleanString);
-};
+export const parseBoolean = booleanString => booleanString.toString().toLowerCase() === 'true';
```

Comment:
Looks like this might be getting called with `undefined` or `null`s some might be best to be even more defensive : `booleanString?.toString?.()...`

Reply:
4f4a5bd17844653800c02cdf9e2d140a4c308c5a.